src/test/resources/sec/cloud.key

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
uff, this is brutal, maybe add this as a secret to the github organisation and then we store the key as a file in github workflow ? Or just pass it as an environment variable to the system?

Reply:
You are right.
I removed the key file.
I added the key as a repository secret, and fed it into the maven build stage in the Github workflow as an environment variable called `VM_PRIVATE_KEY`.
The test reads the private key from that environment variable. In order to allow running the tests locally, you can add a base64 version of the private key as an environment variable directly in the test configuration.
I also annotated the test class in a way that disables it if the environment variable is not available.